hooks/iosAddTarget.js

```diff
@@ -316,6 +320,8 @@ module.exports = function (context) {
             if (productName.indexOf('ShareExt') >= 0) {
               buildSettingsObj['PROVISIONING_PROFILE'] = PROVISIONING_PROFILE;
               buildSettingsObj['DEVELOPMENT_TEAM'] = DEVELOPMENT_TEAM;
+              buildSettingsObj['CODE_SIGN_STYLE'] = 'Manual';
+              buildSettingsObj['CODE_SIGN_IDENTITY'] = '"iPhone Distribution"';
```

Comment:
The rest of the PR looks good to me, but this seems like the above either solves another issue or commit 7d6fa27554bb1980d3985f097462963071962701 shouldn't be part of the PR? Should it?

Reply:
Yes and no. The issue that I was trying to resolve is #82. The issue is mainly building a cordova app in a CI environment so this is needed too, I believe.
I assume that if you are working with XCode you can check the "automatically sign..." but you can't do it from a CI so I think the default should be manual.
I can change the title of the pull request if it makes any difference.
When running Cordova from command line you can specify this in the build arguments but it won't affect this extension.
Might be it should be a feature request to Cordova-ios, not sure, but I've sent them a PR for fixing the problem with provisioning profiles and haven't heard back from them so I don't know...

Reply:
OK, I wonder if `Manual` code sign style will break the build in other cases. Let's merge and see.